libs/agno/agno/eval/performance.py

```diff
@@ -144,7 +144,7 @@ def print_results(self, console: Optional["Console"] = None):
 
 
 @dataclass
-class PerfEval:
+class PerformanceEval:
```

Comment:
This is technically a breaking change, so make sure to mention this in the changelog

Reply:
Yes, good one!